src/__Private/LinterCLI.hack

```diff
@@ -102,12 +111,12 @@ final class LinterCLI extends CLIWithArguments {
     $roots = $this->getArguments();
 
     if (C\is_empty($roots)) {
-      $config = LintRunConfig::getForPath(\getcwd());
+      $config = LintRunConfig::getForPath($this->cwd ?? \getcwd());
```

Comment:
Why do you need this rather than specifying a root?

Reply:
To keep the backward compatibility when `$cwd` is omitted

Reply:
There is a warning if I specify the root

> Warning: PATH arguments contain a hhast-lint.json